plivo/resources/applications.py

```diff
@@ -75,14 +75,12 @@ def create(self,
         if subaccount:
```

Comment:
can we bump the version and add the necessary changes?,
refer [here](https://docs.google.com/document/d/1a_vv-7mr2xmztXHts5l789f9l9vvZIso9IsENLKsYIE/edit?usp=sharing)